doc/languages-frameworks/python.md

```diff
@@ -0,0 +1,736 @@
+# Python
+
+## Tutorial
+
+Several versions of Python are available on Nix as well as a high amount of
+packages. The default interpreter is CPython 2.7.
+
+### Using Python
+
+#### Installing Python and packages
+
+It is important to make a distinction between Python packages that are
+used as libraries, and applications that are written in Python.
+
+Applications on Nix are installed typically into your user
+profile imperatively using `nix-env -i`, and on NixOS declaratively by adding the
+package name to `environment.systemPackages` in `/etc/nixos/configuration.nix`.
+Dependencies such as libraries are automatically installed and should not be
+installed explicitly.
+
+The same goes for Python applications and Python packages that are used as
```

Comment:
This is a tiny bit confusing. Maybe just drop this sentance? Not really sure what information it adds